### PR TITLE
Updated sliding panel

### PR DESCRIPTION
**Pull request by namini67**
This UI change is one which updates the Sliding Panel
The previous functionality was that when the panel was opened, it would slide over the contents of the page. The "non-redux" Sliding Panel component does this still. The Redux Sliding Panel, however, hooks into the Application Layout component and adjusts the size of the Application Content depending on if the Sliding Panel is open or not. This means it "sqeezes" the page layout, to fit the panel on the side.
This functionality is very similar to the current SideNav.

### CODE REVIEW

File: packages/visual-stack-docs/src/containers/App/SlidingPanel.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-alert */
```
Comment (fhwrdh) [inline packages/visual-stack-docs/src/containers/App/SlidingPanel.js]: I believe this line is unnecessary in this file?

File: packages/visual-stack-docs/src/containers/Components/Docs/slidingpanel.js [hunk 2]
```diff
@@ -133,52 +133,12 @@ class VSRSlidingPanelDemo extends React.Component {
...
-                      </VSRSlidingPanelDropdown>
-                    </VSRSlidingPanel>
-                    { /* s6:end */ }
-                    <div>Import Sliding Panel component from visual-stack-redux</div>
-                    <Snippet tag="s5" src={snippets} />
-                    <div>Implementation (similar to presentational components)</div>
-                    <Snippet tag="s6" src={snippets} />
+                    <div>The Redux Sliding Panel must be passed in as a prop to the Application Layout component, similar to the SideNav component.</div>
+                    <div>The Toggle Icon can then be placed where ever in the application you want it to be.</div>
+                    <div>So the recommended usage is to build you Sliding Panel component, then pass it into your Application Layout, so it can slide and not cover your page layout like the non Redux Sliding Panel will.</div>
```
Comment (fhwrdh) [inline packages/visual-stack-docs/src/containers/Components/Docs/slidingpanel.js]: s/build you/build your/

File: packages/visual-stack-docs/src/containers/App/SlidingPanel.js (hunk 1 above)
Comment (namini67) [reply packages/visual-stack-docs/src/containers/App/SlidingPanel.js]: removed it

File: packages/visual-stack-docs/src/containers/Components/Docs/slidingpanel.js (hunk 2 above)
Comment (namini67) [reply packages/visual-stack-docs/src/containers/Components/Docs/slidingpanel.js]: assuming this is just the "you" to "your" which I just updated

**Pull request merged**